src/Resources/Shader.cpp

```diff
@@ -213,6 +213,26 @@ void ShaderProgram::setUniform(const std::string& name, int val) {
   glUniform1i(uniform(name), val);
 }
 
+void ShaderProgram::setVec3(const std::string& name, const glm::vec3& value) { 
```

Comment:
These are just repeats of the above.

void ShaderProgram::setUniform(const std::string& name, const vec3& v) {
  glUniform3fv(uniform(name), 1, value_ptr(v));
}